install-rust.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
```

Comment:
a few more tweaks missing from what I see: both docker/Dockerfile.testing and .devcontainer/Dockerfile need to be adjusted (and probably need to use this script). Also README.md has wrong instructions (should point to use install-rust.sh instead)